File: src/targets/php/http2/fixtures/nested.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('http://mockbin.com/har');
$request->setRequestMethod('GET');
$request->setQuery(new http\QueryString([
  'foo[bar]' => 'baz,zap',
  'fiz' => 'buz',
  'key' => 'value'
]));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/rust/reqwest/fixtures/nested.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "http://mockbin.com/har";

    let querystring = [("foo[bar]", "baz,zap"), ("fiz", "buz"), ("key", "value")];

    let client = reqwest::Client::new();
    let response = client.get(url)
        .query(&querystring)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    dbg!(results);
}
